src/ralph/dc_view/static/js/app/data_center/controllers.js

```diff
@@ -59,7 +59,12 @@
                         rack.new = false;
                         rack.saved = true;
                         $scope.forms.edit_form.$error.all = null;
-                        $scope.forms.edit_form.$success = ['The rack has been successfully added to data center.'];
+                        if (rack.id !== undefined) {
```

Comment:
OK. If so, the line no. 47 (original) should also be changed in the same way (?)